index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Abel Tefera | Software Engineer</title>
+  </head>
+  <body>
+    <header class="header">
+      <a id="logo" class="disable-default" href="#">Abel Tefera</a>
+      <nav class="hamburger">
+        <img src="assets/hamburger.svg" width="30" />
+      </nav>
+      <div class="links-container">
+        <a class="links disable-default" href="#">Portfolio</a>
+        <a class="links disable-default" href="#">About</a>
+        <a class="links disable-default" href="#">Contact</a>
+      </div>
+    </header>
+    <section class="container">
+      <h1 id="headline">I'm Abel, <br />Glad to see you!</h1>
+      <p id="description">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <h5 id="subtitle">LET'S CONNECT</h5>
+      <ul class="social-links">
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/twitter.svg" />
+          </a>
+        </li>
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/medium.svg" />
+          </a>
+        </li>
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/angellist.svg" />
+          </a>
+        </li>
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/github.svg" />
+          </a>
+        </li>
+      </ul>
+      <div class="shapes">
+        <div>
+          <img src="/assets/outer-circle.svg" id="outer-circle-2" />
+          <img src="/assets/rectangle.svg" id="rectangle-1" />
+        </div>
+        <div>
+          <img src="/assets/circle-grey.svg" id="circle-1" />
+          <img src="/assets/rectangle-purple.svg" id="rectangle-2" />
+        </div>
+        <div>
+          <img src="/assets/semi-circle.svg" id="semi-circle-1" />
+          <img src="/assets/outer-circle-grey.svg" id="outer-circle-1" />
+        </div>
+        <div>
+          <img src="/assets/circle.svg" id="circle-2" />
+          <img src="/assets/rectangle-grey.svg" id="rectangle-3" />
+        </div>
+        <div>
+          <img src="/assets/rectangle.svg" id="rectangle-4" />
+          <img src="/assets/semi-circle.svg" id="semi-circle-2" />
+        </div>
+      </div>
```

Comment:
- [ ] You have done a very great job to ensure that the section has a background image 👏 👏 . However, since the image is for decoration purposes, we don't recommend using `img` tag but instead, use the `background` in the `CSS` file ☺️ .  An image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info ☺️ .